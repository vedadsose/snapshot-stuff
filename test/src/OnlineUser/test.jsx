import React from "react";
import UserStatus from "./";
import renderer from "react-test-renderer";

describe("UserStatus", () => {
  describe("when user is online", () => {
    it("renders his status as online", () => {
      const tree = renderer.create(<UserStatus isOnline />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });

  describe("when user is offline", () => {
    it("renders his last seen", () => {
      const tree = renderer
        .create(<UserStatus lastSeen="10 minutes ago" />)
        .toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
