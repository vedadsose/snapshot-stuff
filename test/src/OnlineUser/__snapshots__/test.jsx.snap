// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserStatus when user is offline renders his last seen 1`] = `
<div
  style="color: gray"
>
  <div
    class="icon icon-gray-user"
  />
   Offline since 
  10 minutes ago
</div>
`;

exports[`UserStatus when user is online renders his status as online 1`] = `
<div
  style="color: green"
>
  <div
    class="icon icon-online"
  />
   Online
</div>
`;
